src/client/builtin_scene/client_renderer.cpp

```diff
@@ -289,6 +289,20 @@ namespace builtin_scene
         if (instance.setOpaque(webContentComponent->isOpaque()))
           hasChanged = true;
 
+        {
+          // Update instance for plane dimensions and border radius
+          float width = webContentComponent->logicalWidth();
+          float height = webContentComponent->logicalHeight();
+          float defaultRadius = 20.0f;
+
+          instance.setDimensions(width, height, hasChanged);
+          instance.setBorderRadius(defaultRadius,
```

Comment:
Even though setting all corners to 20.0f, but it seems only the top corners working :(